jkube-kit/resource/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmRepoTypeTest.java

```diff
@@ -78,10 +79,10 @@ public void createConnection_withChartMuseumAndAuth_shouldReturnConnection() thr
   }
 
   @Test
-  public void createConnection_withArtifactoryAndNoAuth_shouldReturnConnection() throws IOException {
+  void createConnection_withArtifactoryAndNoAuth_shouldReturnConnection() throws IOException {
     // When
     final HttpURLConnection result = HelmRepository.HelmRepoType.ARTIFACTORY
-        .createConnection(temporaryFolder.newFile("chart.tar"), helmRepositoryBuilder.build());
+        .createConnection(new File(temporaryFolder, "chart.tar"), helmRepositoryBuilder.build());
```

Comment:
Why didn't you use `File.createTempFile` in this case?

Reply:
I tried but it gives 
```java
java.lang.AssertionError: 
Expecting
  <sun.net.www.protocol.https.DelegateHttpsURLConnection:https://example.com/chart11372277950702050897tar.gz>
to have a property or a field named <"url"> with value
  <https://example.com/chart.tgz>
but value was:
  <https://example.com/chart11372277950702050897tar.gz>
```

Reply:
because your temp file has a wrong extension. The resulting filename should be `chart1833669191392288151.tar.gz` instead of `chart1833669191392288151tar.gz`

I understand that the temp file suffix should be  `.tar.gz` and that you were using `tar.gz`